fastly/resource_fastly_tls_subscription.go

```diff
@@ -294,10 +294,35 @@ func resourceFastlyTLSSubscriptionRead(_ context.Context, d *schema.ResourceData
 	if err != nil {
 		return diag.FromErr(err)
 	}
+
+	// The configuration ID returned by the API is the initial TLS configuration when first
+	// activated. Instead, this value should represent the current TLS configuration id which
+	// can be found via the following API call.
+	// https://api.fastly.com/tls/domains?filter[tls_certificates.id]={cert-id}&include=tls_activations
```

Comment:
I prefer to reference the API documentation...

```suggestion
	// https://www.fastly.com/documentation/reference/api/tls/custom-certs/domains/#list-tls-domains
```